src/utils/events.ts

```diff
@@ -0,0 +1,37 @@
+import { eventsApiUrl, eventsApiKey } from './config';
+
+type EventData = {
+  api_id: string;
+  event: {
+    api_id: string;
+    name: string;
+    start_at: string;
+    cover_url: string;
+    url: string;
+  };
+};
+
+type EventsListData = {
+  entries: EventData[];
+  hasMore: boolean;
+};
+
+export const fetchEventsList = async (limit: number, offset: number): Promise<EventsListData> => {
+  const queryLimit = `pagination_limit=${limit ?? 10}`;
```

Comment:
So far we'll show 10 events on the page. If there is more than 10 - there will be a link to view all events. I think it's fine for MVP